composer.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "xwp/wp-plugin-dependencies",
+    "description": "WordPress plugin dependency management.",
+    "homepage": "https://github.com/xwp/wp-plugin-dependencies",
+    "type": "wordpress-plugin",
+    "license": "GPL-2.0",
+    "version": "1.3.0",
+    "require-dev": {
+        "wp-coding-standards/wpcs": "^0.14.0",
```

Comment:
While I fully support only using stable release, in this case, `develop` would be the branch we need, see my other comment.

Reply:
OK, I just copied this from another `composer.json`.